src/test/java/org/jabref/logic/bibtex/comparator/CrossRefEntryComparatorTest.java

```diff
@@ -3,24 +3,13 @@
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.field.StandardField;
 
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class CrossRefEntryComparatorTest {
-    private CrossRefEntryComparator comparator;
 
-    @BeforeEach
-    public void setUp() {
-        comparator = new CrossRefEntryComparator();
-    }
-
-    @AfterEach
-    public void tearDown() {
-        comparator = null;
-    }
+    private static final CrossRefEntryComparator comparator = new CrossRefEntryComparator();
```

Comment:
Why did you change this? I think it's good practice to initialize the object under test in a setup method, so that e.g. changes to it in one test doesn't leak to other tests.

Reply:
If I see it correctly it is not possible to change the CrossRefEntryComparator Object after creation. In other words there are no setter methods and the constructor does not take any arguments. 

That is why I think the beforeeach is superfluous. Why create a new object, if the one created can be used?

On the other hand, I get your point. If changes to this class are done, like adding arguments to the constructor or adding setter methods, then the BeforeEach makes sense.

Reply:
Creation of objects I java is extremely efficient. This shouldn't be a concern.

Reply:
In tests, one just does not use `static`.

- Possiblity one: class variable (you can go for that, because the `setup` just does that) --> move the initialization code up.
- Possibility two: use `setup` (as in the code)

Reply:
Reason for no static: Each test has to be independed of the last test. Thus, everything under test has to be in the initial state.

Reply:
Thanks for the feedback! I'll revert these changes then and use the setup method for those formatter tests.